src/components/Navigator/NavigatorCard.vue

```diff
@@ -524,7 +384,7 @@ export default {
     apiChanges(value) {
       if (value) return;
       // if we remove APIChanges, remove all related tags as well
-      this.selectedTags = this.selectedTags.filter(t => !this.$t(ChangeNames[t]));
+      this.selectedTags = this.selectedTags.filter(t => !ChangeNameToType[t]);
```

Comment:
```suggestion
      this.selectedTags = this.selectedTags.filter(t => !FILTER_TAGS[t]);
```
Maybe? `selectedTags` should be holding a list of tag types now

Reply:
Edit: I was wrong, `selectedTags` holds a list of tag strings. However, there's no need to convert that name to type again when we can just look up whether a tag exists from `FILTER_TAGS`.
Since `ChangeNameToType` is not used anywhere else, I also removed it from `Changes.js`

Reply:
Edit: I was wrong, `selectedTags` holds a list of tag names (to then be localized). However, there is no need to convert the names back to type again.